src/pages/index.js

```diff
@@ -335,11 +362,133 @@ const IndexPage = (props) => {
                     {contributorCards}
                 </Box>
             </Box>
+            <Box
+                sx={{
+                    padding: theme.spacing(5, 0),
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                }}
+            >
+                <Box
+                    sx={{
+                        padding: theme.spacing(3),
+                        borderRadius: 5,
+                        maxWidth: 'fit-content',
+                        height: 'fit-content',
+                        backgroundColor: 'rgb(218, 209, 198, 0.3)',
+                        display: 'flex',
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                    }}
+                >
+                    <Stack
+                        direction='row'
+                        gap={3}
+                        justifyItems='center'
+                        alignItems='center'
+                    >
+                        <Box
+                            sx={{
+                                display: 'flex',
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                            }}
+                        >
+                            <img
+                                src={thankYou[6]?.replace(/['"]+/g, '')}
+                                alt='Random contributor image'
+                                width={isDesktop ? 100 : isMobile ? 50 : 90}
+                                height={isDesktop ? 100 : isMobile ? 50 : 90}
+                                style={{
+                                    marginTop: 'auto',
+                                    marginBottom: 'auto',
+                                }}
+                            />
+                        </Box>
+                        <Box
+                            sx={{
+                                fontSize: isMobile ? 'small' : 'medium',
+                            }}
+                        >
+                            Thank you{' '}
+                            {thankYou?.filter((item) => item?.trim() === '')
+                                .length === 0 && (
+                                <a
+                                    target='_blank'
```

Comment:
```suggestion
```
`target=_blank` should be avoided: https://css-tricks.com/use-target_blank/

Reply:
That is for opening the link in a new page, I would suggest keeping it.

Reply:
It is a practice in the industry to use `target='_blank'` for external links like this.

Reply:
> That is for opening the link in a new tab, I would suggest keeping it.

To be honest, the piece referenced is highly opinionated and is dated.

Reply:
Added the rel="noreferrer" as suggested in the article cited by @lemeurherve on Gitter https://www.searchenginejournal.com/blank-link-attribute/435883/#:~:text=There%27s%20A%20Security%20Reason%20For,same%20process%20as%20your%20page as a security measure.